web/tools/build.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+#
+# Compile KeymanWeb's dev & test tool modules
+#
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+################################ Main script ################################
+
+# Note:  common/web/recorder is only needed for :recorder, but the builder can't do target-only dependencies yet.
+
+builder_describe "Builds the Keyman Engine for Web's development & unit-testing tools" \
+  "@../../common/web/keyman-version" \
+  "@../../common/web/keyboard-processor" \
+  "@../../common/web/recorder" \
```

Comment:
The following should work for restricting one dependency to `:recorder`:

```suggestion
builder_describe "Builds the Keyman Engine for Web's development & unit-testing tools" \
  "@../../common/web/keyman-version" \
  "@../../common/web/keyboard-processor" \
  "@../../common/web/recorder               :recorder" \
```

(Let me know if this doesn't work -- it's supposed to!)